TinyME-main/src/main/java/ir/ramtung/tinyme/messaging/event/OrderUpdatedEvent.java

```diff
@@ -0,0 +1,15 @@
+package ir.ramtung.tinyme.messaging.event;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Data
+@EqualsAndHashCode(callSuper = false)
+@AllArgsConstructor
+@NoArgsConstructor
+public class OrderUpdatedEvent extends Event {
+    private long requestId;
+    private long orderId;
```

Comment:
All Event subclasses have these two fields. It is better to define these two fields in Event